client/src/common/components/MessageBox.jsx

```diff
@@ -4,7 +4,7 @@ import classNames from 'classnames';
 
 import { MessageType } from 'common/constants/enums';
```

Comment:
Only wonder: will we use the message outside of notifications? Maybe we could unify naming (change message to notification or vice versa)

Reply:
Yes, we  are using them for example when no lists or no cohort are found.